packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -3,10 +3,8 @@
  */
 const { createHash } = require( 'crypto' );
 const path = require( 'path' );
-const { ExternalsPlugin } = require( 'webpack' );
-const { RawSource } = require( 'webpack-sources' );
-// Ignore reason: json2php is untyped
-// @ts-ignore
+const webpack = require( 'webpack' );
+const { RawSource } = webpack.sources || require( 'webpack-sources' );
```

Comment:
How about we document it in the code:
```suggestion
// With webpack 5 it uses `sources` field but for webpack 4 it has to fallback to `webpack-sources` package. 
const { RawSource } = webpack.sources || require( 'webpack-sources' );
```

or something like that.

Reply:
Comment added 🙂